pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/mapper/SegmentCreationMapper.java

```diff
@@ -101,8 +101,7 @@ public void setup(Context context)
     if (readerConfigFile != null) {
       _readerConfigFile = new Path(readerConfigFile);
     }
-
-    _recordReaderPath = _jobConf.get(JobConfigConstants.RECORD_READER_PATH, null);
+    _recordReaderPath = _jobConf.get(JobConfigConstants.RECORD_READER_PATH);
```

Comment:
`RECORD_READER_CLASS_NAME` seems to be more correct name... But, I guess that we are already using this config?

Reply:
Yes, so cannot change it lol